web/src/components/feature/MoleTable.tsx

```diff
@@ -0,0 +1,51 @@
+import { DynamicTable, TDynamicTableInput } from '../common/DynamicTable'
+import { TComponentResponse } from '../../pages/Main'
```

Comment:
This is not exported from Main as far as I know?

Reply:
Fixed. Changes made to Main.tsx was missing. Added now